src/decode.rs

```diff
@@ -2036,6 +2060,7 @@ unsafe fn decode_b(
             }
         }
 
+        let seq_hdr = f.seq_hdr();
```

Comment:
Similar to `f.frame_hdr()`, this can be done at the beginning of `fn decode_b_inner` with a `let seq_hdr = &***f.seq_hdr.as_ref().unwrap();`, so we only ever pay the cost once and it isn't hidden.